packages/test-project/src/App.vue

```diff
@@ -1,16 +1,19 @@
 <template>
   <div id="app">
+    <OldApiExample />
```

Comment:
Let’s start using Vitepress for that maybe? You can see https://github.com/vuelidate/formvuelatte/tree/next for reference. It has it’s problems, but generally works OK. And is amazing for development since the updates are so fast.

Reply:
I have -1 experience with it. But I can try. This means I can drop the whole Vue CLI for the example app.